### PR TITLE
fix: Allow setting db user and password from env variables

### PR DESCRIPTION
**Pull request by lumberj**
See:
https://github.com/interledger/five-bells-shared/blob/master/lib/config.js#L159-L161

### CODE REVIEW

File: src/lib/knex.js [hunk 1]
```diff
@@ -17,8 +17,8 @@ function parseKnexConnection (uri) {
   return {
     host: parsed.hostname,
     port: parsed.port,
-    user: auth[0],
-    password: auth[1],
+    user: auth[0] || config.get('db.connection_user'),
+    password: auth[1] || config.get('database.connection_password'),
```
Comment (MatthewPhinney) [inline src/lib/knex.js]: Why is one `db` and the other `database`?

Comment (lumberj) [inline src/lib/knex.js]: Typo

Comment (lumberj) [inline src/lib/knex.js]: Updated

**Comment by MatthewPhinney** [comment]
LGTM

**Comment by lumberj** [comment]
This method of authenticating to the DB is deprecated

**Pull request closed without merging**